apps/src/templates/BaseDialog.jsx

```diff
@@ -98,7 +98,7 @@ var BaseDialog = React.createClass({
         height: this.props.fixedHeight,
         maxHeight: !this.props.fixedHeight && '80vh',
         overflowX: 'hidden',
-        overflowY: this.props.fixedHeight ? 'hidden' : 'auto',
+        overflowY: (this.props.fixedHeight || this.props.fullHeight) ? 'hidden' : 'auto',
```

Comment:
FullHeight is logically equivalent to fixedHeight in this case (both are component props that indicate a specific height). Including fullHeight in this check to allow scrolling for all base dialogs with a specified height.